appengine/genetics/js/cage.js

```diff
@@ -68,48 +91,55 @@ Genetics.Cage.aliveMice_ = [];
 Genetics.Cage.miceMap_ = {};
 
 /**
- * Mapping of mouse ID to PID of life simulation for that mouse.
- * @type {!Object.<number, number>}
+ * The process ID of the update function for the game logic.
+ * @type {number}
  * @private
  */
-Genetics.Cage.lifePidsMap_ = {};
+Genetics.Cage.pid_ = 0;
 
 /**
  * List of events to be visualized.
  * @type {!Array.<!Event>}
  */
-Genetics.Cage.EVENTS = [];
+Genetics.Cage.Events = [];
+
+/**
+ * Whether events in the Event queue will not be cleared.
```

Comment:
I think this needs more detail. Cleared when; under what circumstances?

Also, maybe try and reword to "will be preserved" or something that is more 'positive', otherwise a dev has to do a little bit of word-acrobatics to flip the boolean.